src/pudl/output/eia860.py

```diff
@@ -11,6 +12,42 @@
 logger = pudl.logging_helpers.get_logger(__name__)
 
 
+@asset(io_manager_key="pudl_sqlite_io_manager", compute_kind="Python")
```

Comment:
I need to update `PudlTabl.pu_eia860()` to read `denorm_utilities_eia860` from the db.